### ISSUE TITLE
`Runtime reflection is not supported` when `buddy/buddy-core` is a dependency

### ISSUE THREAD

**Issue by p-himik** [babashka/tools-deps-native#20]
With this simple `deps.edn`
```clj
{:deps {buddy/buddy-core {:mvn/version "1.10.1"}}}
```
I get:
```
Exception in thread "main" com.oracle.svm.core.jdk.UnsupportedFeatureError: Runtime reflection is not supported for public java.lang.String org.apache.maven.model.Model.getPackaging()
	at com.oracle.svm.core.util.VMError.unsupportedFeature(VMError.java:87)
	at java.lang.reflect.Method.acquireMethodAccessor(Method.java:91)
	at java.lang.reflect.Method.invoke(Method.java:564)
	at org.codehaus.plexus.interpolation.reflection.ReflectionValueExtractor.getPropertyValue(ReflectionValueExtractor.java:331)
	at org.codehaus.plexus.interpolation.reflection.ReflectionValueExtractor.evaluate(ReflectionValueExtractor.java:224)
	at org.codehaus.plexus.interpolation.ObjectBasedValueSource.getValue(ObjectBasedValueSource.java:64)
	at org.codehaus.plexus.interpolation.PrefixedValueSourceWrapper.getValue(PrefixedValueSourceWrapper.java:128)
	at org.codehaus.plexus.interpolation.AbstractDelegatingValueSource.getValue(AbstractDelegatingValueSource.java:44)
	at org.codehaus.plexus.interpolation.StringSearchInterpolator.interpolate(StringSearchInterpolator.java:200)
	at org.codehaus.plexus.interpolation.StringSearchInterpolator.interpolate(StringSearchInterpolator.java:122)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$1.interpolate(StringVisitorModelInterpolator.java:140)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.interpolate(StringVisitorModelInterpolator.java:1455)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:847)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:865)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:865)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:865)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:765)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:664)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:621)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:675)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:264)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator.interpolateModel(StringVisitorModelInterpolator.java:107)
	at org.apache.maven.model.building.DefaultModelBuilder.interpolateModel(DefaultModelBuilder.java:796)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:400)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:252)
	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom(DefaultArtifactDescriptorReader.java:297)
	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.readArtifactDescriptor(DefaultArtifactDescriptorReader.java:175)
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.readArtifactDescriptor(DefaultRepositorySystem.java:255)
	at clojure.tools.deps.alpha.extensions.maven$fn__725.invokeStatic(maven.clj:132)
	at clojure.tools.deps.alpha.extensions.maven$fn__725.invoke(maven.clj:122)
	at clojure.lang.MultiFn.invoke(MultiFn.java:244)
	at clojure.tools.deps.alpha$expand_deps$children_task__418$fn__420$fn__421.invoke(alpha.clj:406)
	at clojure.tools.deps.alpha.util.concurrent$submit_task$task__143.invoke(concurrent.clj:35)
	at clojure.lang.AFn.call(AFn.java:18)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
```

GraalVM used to compile `tools-deps-native`: `graalvm-ce-java11-21.3.0`.
At the same time, `clj -Spath` works just fine, with OpenJDK 11.0.11.

**Comment by borkdude** [comment]
I think @hugoduncan also hit this case and he may already be working on this. What we found was that this happens when the jar file has no `META-INF` or `pom.xml` file inside of it.

**Comment by borkdude** [comment]
Fixed on master, will post here when 0.0.5 is released.

**Comment by borkdude** [comment]
Released.
